tests/default/indices/doc.yaml

```
@@ -57,3 +57,19 @@ chapters:
       id: '1'
     response:
       status: 200
+  - synopsis: Retrieve a non existing document.
```

```suggestion
  - synopsis: Retrieve a nonexistent document.
```

https://english.stackexchange.com/questions/444048/non-existing-or-nonexisting

---

Done